### ISSUE TITLE
VLC Install unresolvable shlib `libx264.so.152'

### ISSUE THREAD

**Issue by Noah-Huppert** [void-linux/void-packages#12463]
### System

* xuname:  `Void 5.0.12_1 x86_64 GenuineIntel notuptodate rrFFFF`
* package: `vlc-3.0.7_1`

### Expected behavior
`xbps-install vlc` should install VLC.

### Actual behavior
```
libavcodec-4.1.3_1: broken, unresolvable shlib `libx264.so.152'
Transaction aborted due to unresolved shlibs.
```

### Steps to reproduce the behavior
```
xbps-install vlc
```

**Comment by Hoshpak** [comment]
Your libx264 is outdated, the current soname is `libx264.so.157`. Please update your system with `xbps-install -Su` after which you will be able to install vlc.

**Comment by Noah-Huppert** [comment]
Thank you @Hoshpak that worked.

I feel like xbps-install not resolving library dependencies is the true unexpected behavior here.  
Resolving dependencies is part of a package managers job. I don't want to have to install a package's dependencies individually before installing the package. Doing that feels more like building a package from source.

**Comment by xtraeme** [comment]
@Noah-Huppert this has been fixed in xbps-0.54 already (git).
